website/docs/r/cognito_identity_pool_provider_principal_tag.markdown

```diff
@@ -50,9 +50,9 @@ resource "aws_cognito_identity_pool_provider_principal_tag" "example" {
 
 The following arguments are supported:
 
-* `identity_pool_id` (Required) - An identity pool ID in the format REGION:GUID.
-* `identity_provider_name` (Required) - The name of the identity provider
-* `principal_tags`: (Optional: []) - String to string map of variables
+* `identity_pool_id` (Required) - An identity pool ID.
```

Comment:
Looking at the provider definition ([here](https://github.com/hashicorp/terraform-provider-aws/blob/78b2dea023f65b5644f783663f32942227e16966/internal/service/cognitoidentity/pool_provider_principal_tag.go#L31-L39)), it links out to an AWS document ([here](https://docs.aws.amazon.com/cognitoidentity/latest/APIReference/API_SetPrincipalTagAttributeMap.html#API_SetPrincipalTagAttributeMap_ResponseSyntax)), which seems to indicate the existing format is correct. Are you seeing differing behavior in practice?